### PR TITLE
small nits

### PR DESCRIPTION
**Pull request by codehag**
I noticed a few small nits while reviewing. Feel free to close if these are not reflecting your intention.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -41,7 +41,7 @@ <h1>Well-known intrinsic objects</h1>
 	<h1>Wrapped Function Exotic Objects</h1>
 	<p>A wrapped function exotic object is an exotic object that wraps a callable object. A wrapped function exotic object is callable (it has a [[Call]] internal method). Calling a wrapped function exotic object generally results in a call of its wrapped function.</p>
 
-	<p>An object is a <dfn id="wrapped-function-exotic-object">wrapped function exotic object</dfn> if its [[Call]] internal method use the following implementations, and its other essential internal methods use the definitions found in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>. These methods are installed in WrappedFunctionCreate.</p>
+	<p>An object is a <dfn id="wrapped-function-exotic-object">wrapped function exotic object</dfn> if its [[Call]] internal method uses the following implementation, and its other essential internal methods use the definitions found in <emu-xref href="#sec-ordinary-object-internal-methods-and-internal-slots"></emu-xref>. These methods are installed in WrappedFunctionCreate.</p>
```
Comment (codehag) [inline spec.html:44]: One comment, "implementation" feels a bit vague for me -- could it be linked?

Comment (leobalter) [reply spec.html:44]: Sure thing! I'll do it in a follow up

File: spec.html [hunk 2]
```diff
@@ -381,9 +381,7 @@ <h1>Runtime Semantics: HostInitializeSyntheticRealm ( _realm_ )</h1>
 		<emu-clause id="sec-hostimportmodulebindingdynamically" aoid="HostImportModuleBindingDynamically">
 			<h1>HostImportModuleBindingDynamically (_callerRealm_, _evalRealm_, _specifier_, _exportName_, _promiseCapability_ )</h1>
 			<p>The host-defined abstract operation HostImportModuleBindingDynamically takes arguments _callerRealm_ (a Realm Record), _evalRealm_ (a Realm Record), _specifier_ (a |ModuleSpecifier| String), _exportName_ (a |IdentifierName| String) and _promiseCapability_ (a PromiseCapability Record). It performs any necessary setup work in order to make available the module corresponding to _specifier_ occurring within the context of the script or module represented by _referencingScriptOrModule_. _referencingScriptOrModule_ is *null*. It then performs FinishDynamicImport to finish the dynamic import process.</p>
-			<p>The implementation of HostImportModuleBindingDynamically must conform with the implementation of HostImportModuleDynamically first and foremost, second, it should conform with the following:</p>
-			<li>
```
Comment (leobalter) [inline spec.html:385]: There is a remaining `</li>` I'm also removing after this, but it should not break rendering for now

**Review by leobalter — APPROVED** [review]
Thank you for the PR, @codehag!

**Pull request merged**